src/app-logic/l10n-bundling.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
A small nit about file name. I think it would be good to make this file name directly l10n instead. It will be especially better after moving the stuff I mentioned in my other comments to this file (since they are not necessarily bundling related).